.drone.yml

```diff
@@ -97,7 +97,7 @@ trigger:
 ---
 kind: pipeline
 type: docker
-name: test-stable
+name: test-stable24
```

Comment:
Shouldn't we test on stable25 at this point? We're already doing it on the app

Reply:
Yes, for sure!